src/po/zh_TW.po

```diff
@@ -1,5275 +1,9128 @@
-# Traditional Chinese Translation for Vim	vim:set foldmethod=marker:
+# Traditional Chinese Translation for Vim
 #
 # Do ":help uganda"  in Vim to read copying and usage conditions.
 # Do ":help credits" in Vim to see a list of people who contributed.
 #
-# FIRST AUTHOR  Francis S.Lin <piaip@csie.ntu.edu.tw>, 2000
 # FIRST RELEASE Thu Jun 14 14:24:17 CST 2001
 #
-# Last update: 2005/01/27 07:03 (6.3)
-#
-# To update, search pattern:		/fuzzy\|^msgstr ""\(\n"\)\@!
-#
-# DO NOT USE WORDS WITH BACKSLASH ('\') AS SECOND BYTE OF BIG5 CHARS
-# EG: '�\', # �\�\�\
-# [blacklist: �\�\�\�\�\�\�\�\�\�\�\�\�\�\�\�\]
-# [blacklist: �\�\�\�\�\�\�\�\�\�\�\�\�\�\]
-# you can replace these characters with alternative words.
-# THIS WILL CAUSE INCOMPATIBLE ON gettext 0.10.36+
-#
-# Note (2005.01.27):
-#  A bug was found for UTF8 mode.
-#   > msgid "%ld fewer lines" "on %ld lines"
-#   If you don't put more (at least 2) spaces after %ld
-#   gvim/win32 will crash (no reason).
-#   So please change [��"] to [�� "]
-#
-# Q. How to use UTF8 mode on Win32?
-# A. A simple configuration: 
-#   set encoding=utf-8; let $LANG='zh_TW.UTF-8'; 
-#   (set langmenu=none or ..)
-#   set termencoding=utf-8
-#   set fileencodings=ucs-bom,utf-8,japan,taiwan,prc
-#   set fileencoding=taiwan (or utf-8)
-#
+# Hung-Te Lin <piaip@csie.ntu.edu.tw>.
+# Cecil Sheng "<b7506022@csie.ntu.edu.tw>.
+# FIRST AUTHOR  Francis S.Lin <piaip@csie.ntu.edu.tw>, 2000.
+# pan93412 <pan93412@gmail.com>, 2018.
 msgid ""
 msgstr ""
-"Project-Id-Version: Vim(Traditional Chinese)\n"
+"Project-Id-Version: Vim (Traditional Chinese)\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2005-01-27 19:00+0800\n"
-"PO-Revision-Date: Mon Feb 19 22:49:21 CST 2001\n"
-"Last-Translator: Hung-Te Lin <piaip@csie.ntu.edu.tw>\n"
-"Language-Team: Hung-Te Lin <piaip@csie.ntu.edu.tw>, Cecil Sheng "
-"<b7506022@csie.ntu.edu.tw>\n"
+"POT-Creation-Date: 2018-07-28 17:13+0800\n"
+"PO-Revision-Date: 2018-07-29 18:15+0800\n"
+"Last-Translator: pan93412 <pan93412@gmail.com>\n"
+"Language-Team: Chinese <zh-l10n@linux.org.tw>\n"
+"Language: zh_TW\n"
 "MIME-Version: 1.0\n"
-"Content-Type: text/plain; charset=big5\n"
-"Content-Transfer-Encoding: 8-bit\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=1; plural=0;\n"
+"X-Generator: Poedit 2.0.9\n"
+
+#: blowfish.c:429
```

Comment:
Please apply `cleanup.vim` after you finish translation.
This kind of comments should be removed by it.

See the README.txt for detail:
https://github.com/vim/vim/blob/fdd7155fab3447b38280035c66178330f8f041e7/src/po/README.txt#L99-L113

Reply:
All the fuzzy & untranslated strings gone when I do `:source cleanup.vim`... I think the PR needs back to status "WIP" -- Not all the strings translated.